### ISSUE TITLE
Sync Back to Bitbucket ?

### ISSUE THREAD

**Issue by regicsolutions** [ef-labs/stash-hook-mirror#86]
I have this working where I am able to mirror my Bitbucket repo to CodeCommit and was wondering if there is a way this plugin can support pulling changes from the remote/upstream repo as well, basically looking to do a git pull —all at the same time…

**Comment by adrianluisgonzalez** [comment]
Not really, it expects the mirror to be readonly.  It works as a bitbucket `PostRepositoryHook` to push changes immediately after commits.  By default it pushes `refs/heads/*` (plus optionally tags and notes).

I suppose you could try a `PreRepositoryHook` to pull changes before the commit, but I think that would lead to conflict resolution problems.
